WTWSMS/common/religions/00_religions.txt

```diff
@@ -1527,6 +1540,7 @@ muslim = {
 	}
 	druze = {
 		graphical_culture = muslimgfx
+		has_coa_on_barony_only = yes
```

Comment:
I removed "kebab" from my pound sign comment in the Religious text, per your request. Is that what you are referring to?